crates/neon-runtime/src/napi/bindings/types.rs

```diff
@@ -50,10 +69,20 @@ pub(crate) type Finalize = Option<
     ),
 >;
 
+#[cfg(feature = "napi-4")]
+pub type ThreadsafeFunctionCallJs = Option<
```

Comment:
This name confused me. Maybe `ThreadsafeFunctionJsCallback`?

Reply:
This is the name that N-API uses. I don't think we should change it unless we have a strong reason:
https://nodejs.org/api/n-api.html#n_api_napi_threadsafe_function_call_js